.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [16.x]
```

Comment:
```suggestion
        node-version: [16.x, lts/*]
```

Reply:
And might want to change line 19 to 
```
        uses: actions/setup-node@v2
```

Reply:
see https://github.com/actions/setup-node/tree/d08cf222111d5c1d21b3cd4b958937f818d10d9a#usage + i tested this so it works :)) 

ref: https://github.com/uclaacm/teach-la-ts-react-starter/pull/122